packages/@react-spectrum/dialog/stories/DialogTrigger.stories.tsx

```diff
@@ -355,6 +358,10 @@ storiesOf('DialogTrigger', module)
         </DialogTrigger>
       </Flex>
     )
+  )
+  .add(
+    'adjustable dialog',
+    () => <AdjustableDialog />
```

Comment:
Noticed the following header and heading collision: 
<img width="639" alt="image" src="https://user-images.githubusercontent.com/9661127/159088231-8799dcb5-c146-49e0-8c86-ac41a5ab46ff.png">